code-of-conduct.md

```diff
@@ -1,113 +1,78 @@
-## Diversity Statement
+## Code of Conduct
+
+The Code of Conduct is designed to protect the US-RSE community and its members from physical and psychological harm.
+To this end, we consider the following behaviors unacceptable in any US-RSE community spaces, in person or online:
+- Trolling, insulting or derogatory comments, and personal attacks.
+- Unwanted physical contact, sexual comments, and/or sharing of sexual material, whether in person or online.
+- Stalking or harassment.
+- Refusing to call someone by their preferred name or pronouns (for example, “dead naming”).
+- Threats or incitement of violence.
+- Behavior that could reasonably be considered inappropriate in a professional setting in the United States.
+- Continuing to communicate with someone after they have asked that they not be contacted.
+- Deliberately disclosing information given in private confidence (for example, “outing”), unless it is to report a violation of this code of conduct (for example, harassment).
+- Deliberate attempts to circumvent these enumerated behaviors or otherwise cause harm to the community or its members.
+
+The Code of Conduct will be enforced by the committee (members enumerated below), which is empowered to respond with sanctions to anyone who conducts themselves in any of these ways, such as:
+- Issuance of formal warnings and possible mediation.
+- Removal of responsibilities, privileges, or access to community spaces.
+- Temporary bans from community spaces.
+- Permanent bans from community spaces.
+
+The Code of Conduct committee will select a Case Lead to serve as a point of contact for a report, and is advised to complete a case within one month when possible.
```

Comment:
Small grammar thing
```suggestion
The Code of Conduct committee will select a Case Lead to serve as a point of contact for a report and is advised to complete a case within one month when possible.
```

Reply:
maybe also instead of "is advised to", this could say "will strive to"